src/snapred/backend/data/LocalDataService.py

```diff
@@ -35,35 +37,58 @@
 """
 
 
+def _createFileNotFoundError(msg, filename):
+    return FileNotFoundError(NOT_FOUND, os.strerror(NOT_FOUND) + " " + msg, filename)
+
+
 @Singleton
 class LocalDataService:
     reductionParameterCache: Dict[str, Any] = {}
     iptsCache: Dict[str, Any] = {}
     stateIdCache: Dict[str, str] = {}
-    dataPath = Config["instrument.home"]
-    instrumentConfigPath: str = dataPath + Config["instrument.config"]
     instrumentConfig: InstrumentConfig  # Optional[InstrumentConfig]
 
-    def __init__(self) -> None:
-        self.instrumentConfig = self.readInstrumentConfig()
+    def __init__(self, verifyPaths: bool = True) -> None:
```

Comment:
I would change this verifyPath to a value on the application.yml and set it to false in test.yml